### ISSUE TITLE
How to precisely emulate the Oculus touch thumbsticks with the Vive controllers' trackpads?

### ISSUE THREAD

**Issue by theSe1f** [LibreVR/Revive#692]
According to this diagram https://images.google.com/imgres?imgurl=https%3A%2F%2Fi.imgur.com%2F5dg0siL.png&imgrefurl=https%3A%2F%2Fgithub.com%2FLibreVR%2FRevive%2Fissues%2F641&docid=HN1BC6eIqNRTiM&tbnid=-QNZ2u1YuD70vM%3A&w=1147&h=1079&source=sh%2Fx%2Fim for emulating the Oculus touch controllers with Vive controllers, emulating the X, Y , A and B Oculus touch buttons are fairly intuitive to emulate with the vive trackpads but I find it  really tricky to accurately emulate the different directions of movement of the Oculus touch thumbsticks accurately eg. With the game Mission:ISS or other tiles utilizing the thumbsticks..  ie. to accurately map moving the virtual thumbsticks on the Vive trackpads precisely up, down, left or right specifically.. Any tips on precisely moving the virtual thumbsticks on the trackpads would be appreciated! It's hard to figure out as according to the diagram the thumbsticks are emulated in the top left semi circle half of the left Vive controller track pad and top right semi circle half of the right Vive controller trackpad respectively but that doesn't seem so intuitive.. Do you tap,  press or slide and in which direction and  in which part of that semi circle do you do that to emulate moving the Oculus thumbstick up, down, left and right respectively? Thank you for any feedback

**Comment by SimplexPL** [comment]
Did you try running the touch tutorial? (not first contact). It shows Touch in you hand and animation of fingers as you press varius buttons and analog sticks. Maybe this will help.

**Comment by theSe1f** [comment]
Yes I have before but will try again..I heard that actually to activate the virtual thumbsticks you need to first tap/lightly touch (not press/click) the top left and right semi circle areas of the two Vive controller trackpads (as in the above diagram) and afterwards maintaining contact with the trackpad you then have the whole circle of the trackpad to move the virtual thumbsticks (not just the top left/right half only).. It would be good for someone to confirm if this is true or not and also if you need to maintain contact do you slide your thumb (lightly touching but not depressing/clicking) in the direction you want to move the virtual thumbsticks in (up, down, left, right etc) or do you have to fully depress/click the trackpad down in the direction you want to move the virtual thumbsticks..? Appreciate any more feedback- thank you
